src/test/resources/Features/OCD-2049-productHistoryEye.feature

```diff
@@ -1,5 +1,5 @@
 Feature: OCD-2049 - Verify 'Eye' on listing details page works and displays Product History 
-
+         OCD-2822 - Add Developer, Product and Version name change to the eye 
```

Comment:
If this feature file now has two tickets in it, the file name should change to indicate that it's not specific to a single feature